### PR TITLE
add services annotations and logging spec to peer and ord

### PR DESCRIPTION
**Pull request by huytran1ibm**
**What this PR does / why we need it**:
This PR addresses 3 issues: 

- Adding annotations to services which are useful by some workloads monitoring the service

- Using fabric logging spec for ord/peer which is the preferred log level mechanism

- Allow the use of initContainer image passthrough for hlf-ca. Useful if cluster does not have access to public image repositories.

**Which issue(s) this PR fixes**:

https://github.com/owkin/charts/issues/45

https://github.com/owkin/charts/issues/44

**Special notes for your reviewer**:

**Checklist**:
- [ X] Bumped chart version in Chart.yaml
- [ X] Added an entry in the CHANGELOG.md
- [ ] Added an entry in the UPGRADE.md if this pull request creates a backward compatibility breaking change
- [ ] Reference your chart in the build matrix of the .travis.yml (For new charts)

### CODE REVIEW

File: hlf-ca/values.yaml [hunk 1]
```diff
@@ -6,8 +6,14 @@ image:
   repository: hyperledger/fabric-ca
   tag: 1.4.3
   pullPolicy: IfNotPresent
+initContainer:
+  image:
+    repository: jwilder/dockerize
+    tag: latest
+    pullPolicy: IfNotPresent
```
Comment (AurelienGasser) [inline hlf-ca/values.yaml:13]: This is not used in the chart

**Review by AurelienGasser — COMMENTED** [review]
Thanks for the PR 🙏

File: hlf-ca/values.yaml (hunk 1 above)
Comment (huytran1ibm) [reply hlf-ca/values.yaml:13]: thanks for catching, fix.

File: hlf-ca/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.2] - 2022-02-17
```
Comment (ClementGautier) [inline hlf-ca/CHANGELOG.md:7]: You can bump to 2.1.0 as this PR adds features to the charts, not only fixing bugs.

File: hlf-peer/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.1] - 2022-02-17
```
Comment (ClementGautier) [inline hlf-peer/CHANGELOG.md:8]: 5.1.0 for the same reasons

File: hlf-couchdb/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.2] - 2022-02-17
```
Comment (ClementGautier) [inline hlf-couchdb/CHANGELOG.md:7]: 2.1.0 for the same reasons

File: hlf-ord/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.2] - 2022-02-17
```
Comment (ClementGautier) [inline hlf-ord/CHANGELOG.md:8]: 3.1.0 for the same reasons

**Review by ClementGautier — CHANGES_REQUESTED** [review]

File: hlf-ord/templates/configmap--ord.yaml [hunk 6]
```diff
@@ -37,6 +37,10 @@ data:
   ORDERER_GENERAL_CLUSTER_ROOTCAS: "/var/hyperledger/tls/server/cert/cacert.pem"
   GODEBUG: "netdns=go"
   ADMIN_MSP_PATH: /var/hyperledger/admin_msp
+  ###########
+  # Logging #
+  ###########
+  FABRIC_LOGGING_SPEC: {{ .Values.logging.spec | quote }}
```
[the remark this answers is not in the document]
Comment (huytran1ibm) [reply hlf-ord/templates/configmap--ord.yaml:43]: image

Good question, i've confirmed that it keeps default behavior.

Left side of ss has FABRIC_LOGGING_SPEC=""
Right side of ss has no FABRIC_LOGGING_SPEC defined (removed from peer env)

File: hlf-ord/CHANGELOG.md (hunk 5 above)
Comment (huytran1ibm) [reply hlf-ord/CHANGELOG.md]: done

File: hlf-peer/CHANGELOG.md (hunk 3 above)
Comment (huytran1ibm) [reply hlf-peer/CHANGELOG.md]: done

File: hlf-ca/CHANGELOG.md (hunk 2 above)
Comment (huytran1ibm) [reply hlf-ca/CHANGELOG.md]: done

**Review by ClementGautier — APPROVED** [review]

**Comment by huytran1ibm** [comment]
@AurelienGasser what now to get this merge?

**Pull request merged**